config/locales/en.yml

```diff
@@ -119,6 +121,9 @@ en:
       city: City
       state: State
       zipcode: ZIP Code
+    errors:
+      invalid_ssn: 'Please enter a valid SSN in the format of 123-45-6789'
```

Comment:
the format specified isn't actually allowed. Only accepts numeric input. Which is fine, but ought to alert the user of that.